retry/get_body_go17.go

```diff
@@ -29,3 +29,7 @@ func getBody(r *http.Request) func() (io.ReadCloser, error) {
 	}
 	return nil
 }
+
+func RemoveGetBody(r *http.Request) {
```

Comment:
This looks like a test helper, so probably shouldn't be part of the package interface.